cli/tests/integration/info_tests.rs

```diff
@@ -1,40 +1,6 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
 use crate::itest;
-use test_util as util;
-use test_util::TempDir;
-
-#[test]
-fn info_with_compiled_source() {
```

Comment:
Why just remove the test... This is an strong indication that we are introducing a breaking change with `deno info`, something we should address.

Reply:
Yes, it was removed because of the breaking change (address in PR description). This is testing if "emit: " is in the output on line 35 and that's no longer the case.